src/invision/components/movie-list/movie-list.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import './movie-list.scss';
+
+export default class MovieList extends React.Component {
+  render() {
+    const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+    return (
+      <section className="movie-list-section">
+        <div className="movie-list-ctnr">
+          <div className="header">
+            <div className="categories">
+              <div className="category">All</div>
+              <div className="category">DOCUMENTARY</div>
+              <div className="category">COMEDY</div>
+              <div className="category">HORROR</div>
+              <div className="category">CRIME</div>
+            </div>
+            <div className="sort">
+              SORT BY
+            <select>
+                <option>RELEASE DATE</option>
+                <option>Category</option>
+              </select>
+            </div>
+          </div>
+          <div className="movie-list">
+            <div className="list-count">
+              39 movies Found
+          </div>
+            <div className="items">
+              {
+                array.map(() => {
+                  return (
+                    <div className="item">
+                      <div className="thumbnail">
+                        <img src={"./thumbnail.png"} alt="film-thumbnail" />
```

Comment:
Your build config does not copy image to the dist directory and app cannot find needed images in the given paths. To solve this problem you can use **file-loader** package - https://webpack.js.org/guides/asset-management/#loading-images